skipchain/skipchain.go

```diff
@@ -68,6 +69,10 @@ type Service struct {
 // added.
 func (s *Service) StoreSkipBlock(psbd *StoreSkipBlock) (*StoreSkipBlockReply, onet.ClientError) {
 	prop := psbd.NewBlock
+	if !s.ServerIdentity().Equal(prop.Roster.Get(0)) {
```

Comment:
as far as I can see, this is not a problem. Changing the roster means a `view-change`. So as long as the verification-functions allow this view-change, that's OK.